cli/flox-rust-sdk/src/models/manifest.rs

```diff
@@ -1551,6 +1572,13 @@ pub(super) mod test {
             #   echo "it's gettin' flox in here"
             # '''
 
+            # The `[services]` section of the manifest allows you to define services.
+            # Services defined here use the packages provided by the `[install]` section
+            # and any variables you've defined in the `[vars]` section or `hook.on-activate` script.
+            [services]
+            # [services.postgres]
+            # command = "postgres --config-file=pg.conf"
```

Comment:
for packages we say 

```toml
[packages]
my-package.pkg-path = ...
```

or

```toml
my-package = { pkg-path = ..., ... }
```

the latter syntax is certainly a bit verbose, but suggesting a third way of defining a struct (while valid toml) seems a bit inconsistent

Reply:
I don't think it reads as well but agree that consistency is better. Will use the former.